src/libAtomVM/term.h

```diff
@@ -1392,12 +1406,47 @@ static inline int term_is_number(term t)
 /**
  * @brief Prints a term to stdout
  *
- * @details Print any given term to the standard output.
+ * @details Print any term to the given file.
+ * @param fd the file where the term will be printed.
```

Comment:
Should there be a deprecation warning in this doc comment?  (not that we read the generated docs, though we do generate them...)

Reply:
I would do the whole deprecation thing on the next PR.